scripts/install.sh

```diff
@@ -7,6 +7,7 @@ DUMP_VERSION='v2'
 # setup
 build_dependencies='curl git build-essential'
 sudo apt-get update > /dev/null
+sudo apt-get upgrade > /dev/null
```

Comment:
Add `-y` to this line to avoid blocking on user input.

Reply:
Yep, got it locally, just need to push it.

Reply:
See 26fc2f1